pkg/operator/utils/utils.go

```diff
@@ -112,6 +138,14 @@ func GetInfrastructure(c client.Client) (*configv1.Infrastructure, error) {
 	return infra, nil
 }
 
+func GetAuth(c client.Client) (*configv1.Authentication, error) {
+	auth := &configv1.Authentication{}
+	if err := c.Get(context.TODO(), types.NamespacedName{Name: "cluster"}, auth); err != nil {
```

Comment:
let's thread through the context for client calls

Reply:
done in upcoming commit